docker/intel-dgpu-driver.Dockerfile

```diff
@@ -6,9 +6,6 @@ FROM ${DTK_AUTO} as builder
 
```

Comment:
Can we change the commit title to

docker: Comments the CSE driver

Currently, the CSE driver is not used, so comment it. 

Could we add more detail about which kernel version this image will be built on and support in the commit log?

Reply:
I will add the kernel version details once we test on 4.12.6

Reply:
Can we change this PR comments to

Commenting out the CSE driver as it is not used.

Reply:
done